### ISSUE TITLE
Improve the example endpoint and maybe join it with the schema one

### ISSUE THREAD

**Issue by pchtsp** [baobabsoluciones/cornflow#513]
**Is your feature request related to a problem? Please describe.**

I don't like the current way the `example` endpoint is implemented for various reasons:

1. You cannot download just one example, you download all of them each time.
2. The instances and their solution are not well organized, they match through the name which is very precarious.
3. The endpoint (example) seems very specific. We could generalize something to be a App-specific endpoint where we have the option of getting (a) the schema, (b) examples, (c) something else, for example, a description of the app, a logo, etc.

**Describe the solution you'd like**

1. There should be two endpoints: one for getting a list of examples for a given App and the other one should download the details of only one example.
2. Instead of downloading Instances and Solutions, one should be able to download Experiments that have a Solution=null for the instances that are not solved.
3. A unified API for dags/Apps.

**Describe alternatives you've considered**

downloading all examples every time I want one example.

**Comment by pchtsp** [comment]
currently one downloads an array of instances and solutions examples that are not linked one with the other. It would be better if the endpoint offers downloading a single json that includes a combination of: (1) instance and (2) its solution (if available).

Something like:
```json
{
"instance": {INSTANCE SCHEMA}, 
"solution": {SOLUTION SCHEMA OR NULL}
}
```
so it's easy to link one and the other.

**Comment by ggsdc** [comment]
Fixed by #525
